taipy/core/config/core_section.py

```diff
@@ -271,7 +271,7 @@ def _from_dict(cls, as_dict: Dict[str, Any], id=None, config: Optional[_Config]
             **as_dict,
         )
 
-    def _update(self, as_dict: Dict[str, Any]):
+    def _update(self, as_dict: Dict[str, Any], default_section=None):
```

Comment:
What is this?
There is no such thing as a `default_section` for unique sections like the CoreSection.

Reply:
This is to match the abstractmethod of the Section

Reply:
ok, thx.